docs/modules/operation/pages/deep-dive/performance-data-collection/resource-types.adoc

```diff
@@ -141,12 +167,19 @@ Replace with the following:
 
 === SiblingColumnStorageStrategy
 
-[options="header, autowidth"]
+[options="header" "autowidth", cols="1,2"]
 |===
-| Parameter           | Description
-| sibling-column-name | Alternate string value to use for index.
-| replace-first       | Regex pattern, replaces only the first match.
-| replace-all         | Regex pattern, replaces all matches.
+| Parameter
+| Description
+
+| sibling-column-name
+| Alternate string value to use for index.
+
+| replace-first
+| Regex pattern, replaces only the first match.
+
+| replace-all
+| Regex pattern, replaces all matches.
```

Comment:
Specifies a global regex replacement to perform against the value of the object pointed to by `sibling-column-name`. Every match in the input will be replaced.

Reply:
```suggestion
| Specifies a global regex replacement to perform against the value of the object that `sibling-column-name` points to.
Every match in the input is replaced.
```